src/hazelcore/Action_Exp.re

```diff
@@ -289,11 +289,13 @@ let mk_syn_text =
   | InvalidTextShape(t) =>
     if (text |> StringUtil.is_empty) {
       let (zhole, u_gen) = u_gen |> ZExp.new_EmptyHole;
-      Succeeded(SynDone((ZExp.ZBlock.wrap(zhole), HTyp.Hole, u_gen)));
+      Succeeded(
+        SynDone((ZExp.ZBlock.wrap(zhole), HTyp.Hole(Some()), u_gen)),
```

Comment:
(see comment below -- my impression was that this should be a None case, not a Some() case because there is no syntactic type hole involved here